sample-status-updater-application/pom.xml

```
@@ -14,13 +14,13 @@
         <dependency>
             <groupId>life.qbic</groupId>
             <artifactId>sample-status-updater-infrastructure</artifactId>
-            <version>1.3.0-SNAPSHOT</version>
+            <version>1.2.4</version>
```

```suggestion
            <version>1.3.0-SNAPSHOT</version>
```
We need to stay with the current version